lib/core/macros.nim

```diff
@@ -175,6 +175,22 @@ proc `[]=`*(n: NimNode, i: BackwardsIndex, child: NimNode) =
   ## set `n`'s `i`'th child to `child`.
   n[n.len - i.int] = child
 
+template `or`*(a,b: NimNode): NimNode =
```

Comment:
I dunno what is so hard about using the same style the whole rest of the file is written in. In other words, write ``a, b`` with a space after the comma. Just like in written English/German/French/etc.